src/Base/Interval.hs

```diff
@@ -143,14 +137,27 @@ infixl 6 |-|
 (|-|) :: Interval -> Interval -> Interval
 int1 |-| int2 = normalizeInterval $ int1 |+| invert int2
 
-jumpIntervalFromNote :: Interval -> Root -> Root
+jumpIntervalFromNote :: Interval -> Note -> Note
 jumpIntervalFromNote (Interval iQual iNum) r =
   let
-    newNote    = nextNthNote (getRoot r) $ iNum - 1
-    currDist   = getPitchClass (rootToPitchClass (rootFrom newNote natural)) - getPitchClass (rootToPitchClass r)
+    newNote    = nextNthLetter (getLetter r) $ iNum - 1
+    currDist   = getPitchClass (noteToPitchClass (noteFrom newNote natural)) - getPitchClass (noteToPitchClass r)
```

Comment:
Redundant parens (I think):
`currDist   = getPitchClass (noteToPitchClass $ noteFrom newNote natural) - getPitchClass (noteToPitchClass r)`
 
I dunno -- just looking a bit schemey

Reply:
^ Nice catch. Didn't even see this, the linter hasn't got me fully conditioned yet ;)